release-calendar/test/setup.ts

```diff
@@ -31,7 +31,7 @@ async function main(): Promise<void> {
     synchronize: true,
     dropSchema: true,
     logging: false,
-  }).catch((error) => {
+  }).catch(error => {
```

Comment:
This undos a prettier fix, which we updated recently. To fix, close and reopen your VSCode.